### DISCUSSION TITLE
Voice cloning clip requirements

### DISCUSSION THREAD

**Discussion by anshulwadhawan** [coqui-ai/TTS#2392] [General Q&A]
I have been trying to do voice cloning using a sample EN voice clip using this:

`tts.tts_to_file("This is voice cloning.", speaker_wav="my/cloning/audio.wav", language="en", file_path="output.wav")
`

I tried to find out the requirements for the sample clip but couldn't. Could someone help me with what format, sample rate, length of the clip and/or other requirements would be optimal for this operation.

Thanks.

**Comment by erogol** [comment]
All that is done automatically. Just give a wav file.
